apps/core/tailwind.config.js

```diff
@@ -104,6 +107,17 @@ module.exports = {
       animation: {
         shimmer: 'shimmer 2s infinite linear',
       },
+      height: {
+        'verticalListShort': '21vh',
+        'verticalListLong': '57vh',
+        '4.5': '1.125rem'
+      },
+      width: {
+        '4.5': '1.125rem'
+      },
+      maxHeight: {
+        'verticalListLong': '57vh',
```

Comment:
I wonder if vh is the best way to represent this. vh can have weird behavior on non-standard window sizes. I feel like a static height that exists in the spacing scale is probably good enough?